ietf-wdm-interface.tree

```diff
@@ -95,7 +95,7 @@ module: ietf-ext-xponder-wdm-if
        |              |  +--ro supported-organizational-modes*
        |              |          -> ../../../../supported-mode/mode-id
        |              +--ro explicit-transceiver-mode-ref?   leafref
-       +--rw current-opt-if-och-mode-params
+       +--rw wdm-if-parameters
```

Comment:
wouldn't be better  current-wdm-if-parameters  ?

Reply:
I would also add something that characterize the fact it represent the configured, chosen mode